homeassistant/components/prusalink/sensor.py

```diff
@@ -79,6 +79,58 @@ class PrusaLinkSensorEntityDescription(
             value_fn=lambda data: cast(float, data["telemetry"]["temp-nozzle"]),
             entity_registry_enabled_default=False,
         ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
+            key="printer.temperature.bed.actual",
+            name="Actual Heatbed Temperature",
```

Comment:
We already have this sensor, as the same data is exposed as `printer.telemetry.temp-bed`

Reply:
The same is true for the nozzle temperature. But since the target temperatures are not exposed through telemetry, I wanted to harmonize the sensor data source.

Removing `printer.telemetry.temp-*` would be a breaking change. I am not sure how to resolve this.

Reply:
It's same data, and removing the other one is breaking, so let's not add it here.

Reply:
I guess removing it could be fine until more Prusa Printers use the Prusalink API and the API is more stable.

Although it would lead to the actual nozzle temperature being available through `printer.telemetry.temp-nozzle` and the target temperature through `printer.temperature.nozzle.target`. And the same for the bed.

Should instead the target temperatures be made available with the `printer.telemetry.temp-target-` prefix?

Reply:
The duplicate values are removed and the target temperatures are moved to `printer.telemetry.*`